docs/source/datasets.rst

```diff
@@ -215,6 +215,19 @@ Vocal Activity
 A time series or sequence of events indicating when singing voice is present in a recording. This type
 of annotation is implicitly available when Vocal F0_ or Vocal Notes_ annotations are available.
 
+.. _stroke-name:
+
+Stroke Name
+^^^^^^^^^^^
+An open "tag" that refers to the original name of the stroke type that is played on a Mridangam instrument.
```

Comment:
Maybe we should do this definition general? I can imagine other datasets with stroke types for other instruments, so simply an open tag with the name of the stroke type could be enough